Dashboard/app/js/lib/views/surveys/question-view.js

```diff
@@ -586,10 +586,11 @@ FLOW.QuestionView = FLOW.View.extend(
       FLOW.selectedControl.set('selectedQuestion', null);
       FLOW.selectedControl.set('dependentQuestion', null);
       FLOW.selectedControl.set('selectedCascadeResource', null);
-      const surveyId = FLOW.selectedControl.selectedSurvey.get('keyId');
-      const validationResult = FLOW.surveyControl.validateSurveyToBePublished(surveyId);
-      FLOW.selectedControl.set('publishingErrors', validationResult);
-
+      if (FLOW.selectedControl.get('publishingErrors')) {
+        const surveyId = FLOW.selectedControl.selectedSurvey.get('keyId');
+        const validationResult = FLOW.surveyControl.validateSurveyToBePublished(surveyId);
+        FLOW.selectedControl.set('publishingErrors', validationResult);
+      }
```

Comment:
I was thinking that rather than validating the publishing at this point, we should ideally be validating the question, no? I guess that is out of scope for these changes but maybe you can leave a TODO comment?